### DISCUSSION TITLE
Load model from csv file

### DISCUSSION THREAD

**Discussion by Pulpas** [MilesCranmer/PySR#350] [Q&A] [answered]
Dear all, 

First of all, thanks to all the contributors to put in place such a nice tool.

I am writing to you after spending some time going through the documentation or previously answered questions, and I did not find any duplicate. Let me know if this has already been solved, anyway, here is my question.

I would like to know if it is possible to load a model previously obtained using the function "PySRRegressor.from_file()" not from a .pkl but from a .csv file ? I unfortunately got my job killed for an overload of memory, so I do not have a .pkl file from the which I could restart.

If not possible, is it possible to generate a pkl file from a csv file ?

**Comment by MilesCranmer** [comment, marked as answer]
You should be able to just pass the `.csv` file to `from_file`. But note that the `pkl` file contains some of the settings, so if you are only passing the `.csv` file, you will have to pass some other settings. The API reference for this function is here: https://astroautomata.com/PySR/api/#pysr.sr.PySRRegressor.from_file

**Reply by Pulpas** [reply]
Thanks for your very fast answer, and also pointing the astroautomata guideline, I was not aware of this website (it works like a charm)!
